database.go

```diff
@@ -8,37 +8,13 @@ import (
 	"github.com/jmoiron/sqlx"
 )
 
-const (
-	schema = `CREATE TABLE visits (
-		id int(11) NOT NULL AUTO_INCREMENT,
-    	ip varchar(255) NOT NULL,
-    	host text NOT NULL,
-      user_agent text NOT NULL,
-    	path text NOT NULL,
-		created_at datetime NOT NULL,
-		PRIMARY KEY (id)
-	) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;`
-	checkIfSchemaExists = `SELECT COUNT(*) as does_exist FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'visits'`
-	insertVisit         = `INSERT INTO visits (ip, host, path, user_agent, created_at) VALUES (:ip, :host, :path, :user_agent, :created_at)`
-)
+const insertVisit = `INSERT INTO visits (ip, host, path, user_agent, created_at) VALUES (:ip, :host, :path, :user_agent, :created_at)`
 
 var db *sqlx.DB
 
-type TableCheck struct {
-	DoesExist int `db:"does_exist"`
-}
-
 func init() {
 	db = sqlx.MustConnect("mysql", os.Getenv("PING_DB"))
 	db.Ping()
-	var check TableCheck
-	err := db.Get(&check, checkIfSchemaExists)
-	if err != nil {
-		panic(err)
-	}
-	if check.DoesExist < 1 {
-		db.MustExec(schema)
-	}
```

Comment:
It can be put into its own package to make this easier to share. Perhaps we can move `database.go` into its own `database` package and have this initialization code in there too. 😄 

Then the binary just does `import "github.com/parkr/ping/database"` and `conn := database.Connect(); database.InitializeTable(conn)` or something 😄